sonar-project.properties

```diff
@@ -0,0 +1,2 @@
+sonar.exclusions=.github/**, .vscode/**, android/**, assets/**, build/**, ios/**, node_modules/**, scripts/**
```

Comment:
@typelogic why this modification as part of this PR?

Reply:
i didn't intend to change this one.. i'll clean this up...